Rarime/Code/Models/Passport.swift

```diff
@@ -178,6 +178,17 @@ struct Passport: Codable {
         let encoder = JSONEncoder()
         return try encoder.encode(self)
     }
+    
+}
+
+func deepCopy<T: Codable>(_ object: T) -> T? {
+    do {
+        let data = try JSONEncoder().encode(object)
+        return try JSONDecoder().decode(T.self, from: data)
+    } catch {
+        print("Copy failed: \(error)")
```

Comment:
let's better add a static method `copy(_ other: Passport)` to `Passport` struct and fill the values similar to `fromNFCPassportModel`